mycroft/messagebus/load_config.py

```diff
@@ -0,0 +1,34 @@
+from collections import namedtuple
```

Comment:
Add license block

Reply:
I am going to have to get used to doing this :)